deployment/GI02HYPM_Deploy.csv

```diff
@@ -35,21 +35,21 @@ AR21,,AR30-03,,GI02HYPM-WFP02-04-CTDPFL000,4,1,2017-08-06T13:45:00,2018-06-16T06
-AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA009,6,1,,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55069,,,,,2668,upward-looking
-AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA010,6,1,,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55066,,,,,2668,downward-looking
+AR30-03,,AR35-05,,GI02HYPM-RIM01-00-SIOENG000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGCON-SIOCON-13993,59.96974,-39.49005,,2585,2669,
+AR30-03,,AR35-05,,GI02HYPM-RIM01-02-CTDMOG039,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-CTDMOG-13621,59.96974,-39.49005,,138,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-00-WFPENG000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGCON-GWFPGX-10401,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-01-FLORDL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-FLORDL-03280,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-03-DOSTAL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-DOSTAL-01476,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-04-CTDPFL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-CTDPFL-00131,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-WFP02-05-VEL3DL000,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,CGVEH-GWFPGX-10401,CGINS-VEL3DL-01109,59.96974,-39.49005,,2573,2669,
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA009,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55109,59.96974,-39.49005,,,2669,upward-looking
+AR30-03,,AR35-05,,GI02HYPM-MPM01-02-ZPLSGA010,5,1,2018-06-09T12:55:00,2019-08-10T06:34:00,CGMGI-02HYPM-00005,,CGINS-ZPLSGA-55111,59.96974,-39.49005,,,2669,downward-looking
+AR35-05,,,,GI02HYPM-RIM01-00-SIOENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGCON-SIOCON-12368,59.97287,-39.53082,,2585,2664,
+AR35-05,,,,GI02HYPM-RIM01-02-CTDMOG039,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-CTDMOG-11611,59.97287,-39.53082,,138,2664,
+AR35-05,,,,GI02HYPM-WFP02-00-WFPENG000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGCON-GWFPGX-77401,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-01-FLORDL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-FLORDL-02350,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-03-DOSTAL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-DOSTAL-01106,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-04-CTDPFL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-CTDPFL-00097,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-WFP02-05-VEL3DL000,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,CGVEH-GWFPGX-77401,CGINS-VEL3DL-01025,59.97287,-39.53082,,2573,2664,
+AR35-05,,,,GI02HYPM-MPM01-02-ZPLSGA009,6,1,2019-08-06T12:37:00,,CGMGI-02HYPM-00006,,CGINS-ZPLSGA-55069,59.97287,-39.53082,,,2664,upward-looking
```

Comment:
For the newly deployed HYPM-6 at Location A...
Both ZPLSG instruments should have a deployment depth of 150 m.
WFP instruments should have a deployment depth of 2560 m.
The CTDMO should have a deployment depth of 160 m.
The SIOCON should have a deployment depth of 2570 m.

Reply:
Oh ok - your email said that all other depths should be correct.  Will fix.

Reply:
fixed